target/linux/ramips/mt7621/base-files/lib/preinit/04_handle_checksumming

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
I haven't checked explicitly, but I'd expect that preinit files (in /lib folder) are sourced and thus the shebang should be removed.